tst/EnergyPlus/unit/Datasets.unit.cc

```diff
@@ -0,0 +1,159 @@
+// C++ Headers
+#include <fstream>
+#include <string>
+#include <vector>
+#include <sstream>
+
+// Google Test Headers
+#include <gtest/gtest.h>
+
+// Fixtures, etc.
+#include "Fixtures/EnergyPlusFixture.hh"
+
+namespace EnergyPlus {
+
+	typedef EnergyPlusFixture DataSetFixture;
+
+	std::vector<std::string> getAllLinesInFile( std::string filePath ) {
```

Comment:
@mbadams5 Did you already implement a way to pass a file path to the process_idf function?  I thought you had, but I couldn't find the declaration.  If not, is there a better way to get the lines of the file to pass into process_idf?